### ISSUE TITLE
PHPStorm is outdated by 10 months or so

### ISSUE THREAD

**Issue by mastacheata** [Claud/chocolatey-packages#7]
It's only fair to give you the chance to update it yourself, but you might find these little helpers I built today to be helpful:
https://github.com/mastacheata/chocolatey-packages/blob/master/phpstorm/CheckBuildUpdate.ps1
(And here's the rest with templates, installer scripts and updated nuspec
https://github.com/mastacheata/chocolatey-packages/tree/master/phpstorm)

I've built a PowerShell (3+) script to automatically get the latest version, download link and sha256 checksum from JetBrains and then update the NuSpec and PowerShell install scripts.
It will then run the choco pack command with the updated nuspec file and create a nupkg
Last but not least it could also push that to chocolatey, but for now that option is commented out in the script as I didn't want to create another slug for the same software that's already up there.

Basically that script can be adjusted to suit any JetBrains IDE in less than 5 minutes, but PHPStorm and RubyMine were the only outdated ones.

**Comment by Spunkie** [comment]
@Claud Phpstorm is still very out of date, do you have any intention of updating it?

**Comment by Spunkie** [comment]
@mastacheatav If I start the chocolatey pkg triage process to request a new maintainer would you be willing to being the new maintainer?

**Comment by mastacheata** [comment]
Sure, for the most of it my scripts should already cover auto-updating, so there's not that much work associated with maintaining it, hopefully.

**Comment by Spunkie** [comment]
@mastacheata Ok I've been made a maintainer of the pkg so I can also add you now. 

We should probably move any further discussion into the RFM issue though. Can you please comment in [this issue](https://github.com/chocolatey/chocolatey-package-requests/issues/104) with a link to your chocolatey profile?
